### PR TITLE
Create home page

### PR DESCRIPTION
**Pull request by dominikslawkowski**
Closing #45

<h1 id="preview">Preview</h1> 

image

### CODE REVIEW

**Review by mecies — APPROVED** [review]
Good Job, nothing to complain about 🤣  ✅

**Review by mecies — APPROVED** [review]

**Review by KlaraGajaszek — APPROVED** [review]

File: src/views/about/data.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { CardProps } from '../../components/DetailsCard/cardProps';
+import avatar from '../../images/avatar.jpg';
+import project from '../../images/download.jpeg';
+
+export const projectData = {
```
Comment (KonradSzwarc) [inline src/views/about/data.ts]: What is the purpose of `projectData` and `userData` if they aren't used anywhere?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,22 +1,8 @@
...
-    <div style={{ maxWidth: `300px`, marginBottom: `1.45rem` }}>
-      <Image />
-    </div>
-    <Link to="/page-2/">Go to page 2</Link> <br />
-    <Link to="/using-typescript/">Go to "Using TypeScript"</Link>
-  </Layout>
-)
-
-export default IndexPage
+export default index;
```
Comment (KonradSzwarc) [inline src/pages/index.tsx]: Isn't it possible to write just `export { default } from '../views/about'`?

**Review by KonradSzwarc — CHANGES_REQUESTED** [review]

File: src/pages/index.tsx (hunk 2 above)
Comment (dominikslawkowski) [reply src/pages/index.tsx]: good point 👍

**Review by KonradSzwarc — APPROVED** [review]

**Pull request merged**